Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -315,6 +319,17 @@ public void serverStatus(@Nullable final AnalysisStatus analysisStatus, @Nullabl
     }
   };
 
+  public static boolean doEnableAnalyzedFilesSubscription() {
+    PropertiesComponent properties = PropertiesComponent.getInstance();
```

Comment:
`PropertiesComponent.getInstance()` returns app-level component. If you want this setting to be per-project (I think so reading the title) use `PropertiesComponent.getInstance(Project)` (you'll have to make it not static then).

Cosmetics: how about renaming this method to `isAnalyzedFilesSubscriptionEnabled()`? BTW do you need it to be public?